src/Tables/molecules/TypeTable.tsx

```diff
@@ -80,10 +106,21 @@ const TypeTable : React.FC<IProps> = ({
           {hasStatus ? <HeaderItem fixedWidth={10} /> : null}
           {hasThumbnail ? <HeaderItem fixedWidth={70} /> : null}
           {hasTypeIcon ? <HeaderItem fixedWidth={35} /> : null}
-
           {columnConfig.map((column, key) => {
-            const {alignment, header, hasCopyButton} = column;
-            return <HeaderItem key={key} alignment={alignment} hasCopyButton={hasCopyButton}>{header}</HeaderItem>;
+            const {alignment, hasCopyButton, sortActive, ascending } : ITableColumnConfig = column;
+            return <HeaderItem
+                      key={key}
```

Comment:
IMO this key should use `columnId` by default and fall back to index for columnID

Reply:
I'm not sure, that is the part I really struggled with.
Like, since columnId is optional, can some columns have it and some don't?
Can this cause handling problems?


What I understood from our last conversation was that the UI Library should not work around the Id's that expects from the outside. That is why I went with index as Id's at the end for the management of the UI and made optional to return the columnId in case the it was sent because I'm just a messenger of that data 🚚